scheduler_error_mailer/tests/test_scheduler_error_mailer.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Akretion - Chafique Delli
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields
+from odoo.tests.common import SavepointCase
+
+
+class TestSchedulerErrorMailer(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.test_scheduler_error_mailer = cls.env.ref(
+            "scheduler_error_mailer.test_scheduler_error_mailer"
+        )
+
+    def test_scheduler_failure(self):
+        try:
+            self.test_scheduler_error_mailer.method_direct_trigger()
```

Comment:
This line triggers the job directly and bypasses `_callback` of ir_cron. That is the reason why `_handle_callback_exception` was not fired while running test thus travis build failed. Other than this test, the `ir_cron` code does work.